tests/js/spec/views/eventsV2/savedQueryButtonGroup.spec.jsx

```diff
@@ -2,11 +2,12 @@ import React from 'react';
 import {mountWithTheme} from 'sentry-test/enzyme';
 import {browserHistory} from 'react-router';
 
-import EventSaveQueryButton from 'app/views/eventsV2/saveQueryButton';
+import SavedQueryButtonGroup from 'app/views/eventsV2/savedQueryButtonGroup';
 import {ALL_VIEWS} from 'app/views/eventsV2/data';
 import EventView from 'app/views/eventsV2/eventView';
 
-describe('EventsV2 > SaveQueryButton', function() {
+// eslint-disable-next-line jest/no-disabled-tests
+describe.skip('EventsV2 > SaveQueryButtonGroup', function() {
```

Comment:
@leedongwei hopefully we can restore and polish these tests up soon in a follow up PR 👍

Reply:
Yep, I'm following up with tests in a separate PR but that's probably coming later in the afternoon.